oktoberfest/data/spectra.py

```diff
@@ -201,23 +249,22 @@ def write_pred_as_hdf5(self, output_file: Union[str, Path]) -> Thread:
         return hdf5.write_file(data_sets, output_file, data_set_names, column_names)
 
     @classmethod
-    def from_hdf5(cls: Type[SpectraT], input_file: Union[str, Path]) -> SpectraT:
+    def from_hdf5(cls: Type[SpectraT], input_file: Union[str, Path]):
```

Comment:
same as for write, you can simply use a one-liner:  https://anndata.readthedocs.io/en/latest/generated/anndata.read_hdf.html

Reply:
The read_hdf function only lets you select one dataset with a key so it doesn't work for our purpose. I used this function https://anndata.readthedocs.io/en/latest/generated/anndata.read_h5ad.html as it reads in the whole AnnData Object